test/gie/4D-API_cs2cs-style.gie

```diff
@@ -215,6 +215,32 @@ accept      487147.594520173    4934316.46263998    0
 expect      -10370728.80        5552839.74          0
 -------------------------------------------------------------------------------
 
+-------------------------------------------------------------------------------
+Test that Google's Web Mercator with +proj=merc +aux_sphere_type=0
+-------------------------------------------------------------------------------
+operation   proj=pipeline step init=epsg:26915 inv step proj=merc datum=WGS84 aux_sphere_type=0
+-------------------------------------------------------------------------------
+tolerance   20 cm
+accept      487147.594520173    4934316.46263998
+expect      -10370728.80        5552839.74
+
+accept      487147.594520173    4934316.46263998    0
+expect      -10370728.80        5552839.74          0
+-------------------------------------------------------------------------------
+
+-------------------------------------------------------------------------------
+Test that Google's Web Mercator with +proj=webmerc
+-------------------------------------------------------------------------------
+operation   proj=pipeline step init=epsg:26915 inv step proj=webmerc datum=WGS84
+-------------------------------------------------------------------------------
+tolerance   20 cm
+accept      487147.594520173    4934316.46263998
+expect      -10370728.80        5552839.74
+
+accept      487147.594520173    4934316.46263998    0
+expect      -10370728.80        5552839.74          0
+-------------------------------------------------------------------------------
+
```

Comment:
How about some tests for `aux_sphere_type` == 1 and 2?

Reply:
Added, by the way while adding them I had originally issues with the z component being modified, which I think was wrong. I fixed that issue with commit https://github.com/OSGeo/proj.4/pull/925/commits/1a575b61a18eb480d4482c843e4a42db66c81ba5 which I believe is a bugfix that should probably be applied to the 5.0 branch

Reply:
> Added, by the way while adding them I had originally issues with the z component being modified, which I think was wrong. 

Do you mind writing that up in an issue so it can be referred to in the release notes later on? 

> I fixed that issue with commit 1a575b6 which I believe is a bugfix that should probably be applied to the 5.0 branch

I don't plan on doing more bug fix release for the 5.0 branch. 5.1 will be out in two months. I think that will be fine.